sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_registry_client.py

```diff
@@ -28,24 +28,27 @@
 
 class ContainerRegistryClient(ContainerRegistryBaseClient):
     def __init__(
-        self, endpoint: str, credential: "Optional['AsyncTokenCredential']" = None, *, audience: str, **kwargs: "Any"
+        self,
+        endpoint: str,
+        credential: "Optional['AsyncTokenCredential']" = None,
+        audience: Optional[str] = AZURE_RESOURCE_MANAGER_PUBLIC_CLOUD,
```

Comment:
This should also remain keyword-only, and type is `str`.

Reply:
Since it's already in `kwargs`, it's not necessary to move to `params`. Other than that, I don't see the benefit of using audience in keyword here. We can discuss it offline.

Reply:
It's good to learn the reason behind the general rule to use positional parameters and keyword arguments, thanks Anna! :)